core/src/test/scala/com/salesforce/op/stages/impl/insights/RecordInsightsLOCOTest.scala

```diff
@@ -466,4 +427,167 @@ class RecordInsightsLOCOTest extends FlatSpec with TestSparkContext {
     assertAggregatedTextMap(textAreaMap, "k1")
 
   }
+  it should "aggregate values for date, datetime, dateMap and dateTimeMap derived features" in {
```

Comment:
sorry, I am not able to follow these tests anymore.

Can we please make them more readable by either splitting them into smaller test cases or rather create a behavior (http://www.scalatest.org/user_guide/sharing_tests), or at least add clues `withClue` - http://www.scalatest.org/user_guide/using_assertions

Reply:
sure, will refractor.